packages/arb-token-bridge-ui/src/hooks/useTokenDecimals.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react'
+import { ArbTokenBridge } from 'token-bridge-sdk'
+
+const useTokenDecimals = (
```

Comment:
Not quite convinced we should keep this hook, we can always just inline `selectedToken?.decimals ?? 18`

Reply:
We can't rely on `selectedToken`, for example in withdrawal/deposit panel, if we change selectedToken, we still want to format the number that was deposited/withdrawn.

We need to rely on the address and get the token from `bridgeTokens`
`bridgeTokens?.[tokenAddress]?.decimals ?? 18`

We could move use `useArbTokenBridge` rather than passing `bridgeTokens` as a parameter. So we "encapsulate" `bridgeTokens` in the hook